digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/InaccurateAssetDAO.scala

```diff
@@ -0,0 +1,46 @@
+package fi.liikennevirasto.digiroad2.dao
+
+import fi.liikennevirasto.digiroad2.asset.AdministrativeClass
+import slick.driver.JdbcDriver.backend.Database.dynamicSession
+import slick.jdbc.StaticQuery.interpolation
+
+class InaccurateAssetDAO {
+
+  def createInaccurateAsset(assetId: Long, typeId: Int, municipalityCode: Int, administrativeClass: AdministrativeClass) = {
+    sqlu"""
+        insert into inaccurate_asset (asset_id, asset_type_id, municipality_code, administrative_class)
+        values ($assetId, $typeId, $municipalityCode, ${administrativeClass.value})
+      """.execute
+  }
+
+  def getInaccurateAssetById(assetId: Long): Option[Long] = {
+    sql"""select asset_id from inaccurate_asset where asset_id= $assetId""".as[Option[Long]].firstOption.getOrElse(None)
```

Comment:
Can't we just use `.as[Option[Long]].first`

Reply:
Or `as[Long].firstOption` because `asset_id` is not nullable. And of course remove `getOrElse(None)`